### PR TITLE
docs: add Air-gapped upgrade documentations.

### PR DESCRIPTION
**Pull request by ruiyaoOps**
Signed-off-by: yaorui <yaorui@yunify.com>

### CODE REVIEW

File: content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+   ```
+
+   If you cannot access the object storage service of Google, run the following command instead to add the environment variable to change the source.
+
+   ```bash
+   export KKZONE=cn;./offline-installation-tool.sh -b -v v1.17.9 
+   ```
+
+   {{< notice note >}}
+
```
Comment (pixiake) [inline content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md:141]: For cross-version upgrades, an intermediate version also needs to be downloaded in advance.

Comment (ruiyaoOps) [reply content/en/docs/upgrade/Air-Gapped-Upgrade-with-KubeKey.md:141]: In Prerequisites, I have additional tips about the upgrade to v2.1.1.

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**